package.json

```diff
@@ -32,8 +28,8 @@
   "dependencies": {
     "binary-search-bounds": "^2.0.3",
     "clamp": "^1.0.1",
-    "color-id": "^1.0.0",
-    "font-atlas-sdf": "^1.2.0",
+    "color-id": "^1.1.0",
+    "font-atlas-sdf": "^1.3.1",
```

Comment:
Judging from 

https://github.com/dfcreative/font-atlas-sdf/commits/master

this should be `^1.3.2`, right?